pynanoleaf/pynanoleaf.py

```diff
@@ -40,6 +52,10 @@ class _Info(dict):
     firmwareVersion = _dictitem_property('firmwareVersion')
     model = _dictitem_property('model')
     name = _dictitem_property('name')
+    manufacturer = _dictitem_property('manufacturer')
```

Comment:
That is an awesome change, could you please do this in a separate PR? Makes it easier to review :)

Reply:
ok